src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,284 @@
+    private static String WRONG_COMMAND_ERROR_MESSAGE = SEPARATION_LINE + "\n" + SHORT_SPACE +
+            "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_TODO_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a todo cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_DEADLINE_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a deadline cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_EVENT_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a event cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_DELETE_INDEX_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The index of the deletion cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String WRONG_DELETE_INDEX_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The index of the deletion is out of bound.\n" + SEPARATION_LINE + "\n";
+    private static String ADD_TASK_MESSAGE = SHORT_SPACE + "Got it. I've added this task:";
+    private static String REMOVE_TASK_MESSAGE = SHORT_SPACE + "Noted. I've removed this task:";
+    private static String EXIT_MESSAGE = SEPARATION_LINE + "\n" + "     Bye. Hope to see you again soon!\n" +
+            SEPARATION_LINE;
+
+    private static char NOT_DONE_SYMBOL = '\u2718';
+
+    public static void main(String[] args) {
+        File file = new File("duke.txt");
+        try {
+            file.createNewFile();
+            readFileContents("duke.txt");
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        greet();
+
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        userInput = in.nextLine();
+
+        while (true) {
+            try {
+                processCommand(userInput);
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+            } finally {
+                userInput = in.nextLine();
+            }
+        }
+    }
+
+    static void processCommand(String userInput) throws DukeException {
+        String command = userInput.split(" ")[0];
+
+        switch (command) {
+        case "todo":
+            addTodo(userInput);
+            break;
+        case "deadline":
+            addDeadline(userInput);
+            break;
+        case "event":
+            addEvent(userInput);
+            break;
+        case "done":
+            setTaskToBeDone(userInput);
+            break;
+        case "bye":
+            exit();
+            break;
+        case "list":
+            displayList();
+            break;
+        case "delete":
+            deleteTask(userInput);
+            break;
```

Comment:
clean, may consider having the commands into their own constants

Reply:
Thank you for your compliment! :)